releases/nightly-build/compose-files/README.md

```diff
@@ -0,0 +1,44 @@
+| docker-compose-nexus-add-security.yml | Security **extending** compose file.<br />Adds the additional security services and configuration of services so that all the services are running in the secure configuration. |
+| docker-compose-nexus-add-mqtt.yml     | MQTT **extending** compose file.<br />Adds a MQTT Broker and additional configuration of services so that the `MQTT` implementation of the Edgex Message Buss is used.<br />Also adds the Device MQTT device service which is of course dependent on a MQTT Broker. |
+| docker-compose-nexus-ui.yml           | Stand-alone compose file for running the optional EdgeX UI. Runs in `host` network mode and only supports connecting to local Edgex services via 127.0.0.1 IP address. |
+
+### Makefile
+
+This folder contains a `Makefile` that provides helper commands for running, stopping and cleaning the various EdgeX configurations.
+
+| Command                  | Description                                                  |
+| ------------------------ | ------------------------------------------------------------ |
+| make portainer           | Runs **`Portainer`** independent of the EdgeX services       |
+| make portainer-down      | Stops **`Portainer`** independent of the EdgeX services      |
+| make run                 | Runs the EdgeX services in **secure mode**                   |
+| make run-no-sec          | Runs the EdgeX services in **non-secure mode**               |
+| make run-mqtt            | Runs the EdgeX services using MQTT Message Buss in **secure mode**, including the Device MQTT service. |
+| make run-mqtt-no-sec     | Runs the EdgeX services using MQTT Message Buss in **non-secure** mode, including the Device MQTT service. |
+| make run-ui              | Runs the optional EdgeX UI service.                          |
+| make run-arm             | Runs the **ARM64** EdgeX services in **secure mode**         |
+| make run-arm-no-sec      | Runs the **ARM64** EdgeX services in **non-secure mode**     |
+| make run-mqtt-arm        | Runs the **ARM64** EdgeX services using MQTT Message Buss in **secure mode**, including the Device MQTT service. |
+| make run-mqtt-arm-no-sec | Runs the **ARM64** EdgeX services using MQTT Message Buss in **non-secure** mode, including the Device MQTT service. |
+| make run-ui-arm          | Runs the optional **ARM64** EdgeX UI service.                |
+| make down                | Stops all EdgeX no matter which configuration they are running in. |
+| make clean               | Runs **`down`** and then removes any stopped containers, prunes unused volumes and networks |
+| make remove              | Runs **`clean`** and then removes all docker images with the **`master`** tag. This is how you make sure you are using the very latest **`master`** images. It is highly recommended to run this command prior to any validation of you current pull request. |
```

Comment:
It is highly recommended *you* run this command prior to any validation of *your* current pull request.

Reply:
fixed